Acd 894 (#82)
Actindo-AG/Documentation: Adyen/Overview/01_General.md

# General information

With the help of the Adyen&trade; payment platform, you can offer your customers all of their preferred payment methods - with the help of just one payment service provider. The integration to the *Payments* module gives you a complete overview of all payments made via Adyen&trade; from all sales channels and allows you to easily transfer them to the *Accounting* module. 

> 

Reviewer: Vielleicht erwähnen, dass die Übertragung von Zahlungen aus Payments zu Buchhaltung via Zahlungsabwicklung geht? Sonst hört es sich so an, als würde es direkt aus Payments zu Buchhaltung erfolgen.

> The payment service provider Adyen&trade; provides merchants a single platform to accept payments, protect revenues, and control finances. It offers an all-in-one online payment solution built for business, which means that all the different ways to pay online are supported, such as credit cards, invoice, prepayment.<!---Stefan, stimmt das prepayment--> It integrates popular payment service providers such as Amazon Pay&trade;, Paypal&trade; and much more.

Reviewer: ```suggestion
The payment service provider Adyen&trade; provides merchants with a single platform to accept payments, protect revenues, and control finances. It offers an all-in-one online payment solution built for business, which means that all the different ways to pay online are supported, such as credit cards, invoice, or prepayment.<!---Stefan, stimmt das prepayment--> It integrates popular payment service providers such as Amazon Pay&trade;, Paypal&trade; and many more.
```

This documentation refers to the Adyen&trade; plugin version 1.1.0. There may be differences in the user interface, features, and operation if your Adyen&trade; plugin is installed in a different version.

## Benefits

> - Integration with only one payment service provider that covers all required payment methods.

Reviewer: ```suggestion
- Integration with only one payment service provider that covers all required payment methods
```

- Connection via the *Payments* module

> - Receipt of authorizations, captures, refunds, cancellations, and chargebacks via push notification from Adyen&trade; triggered by external sources (for example, store).

Reviewer: ```suggestion
- Receipt of authorizations, captures, refunds, cancellations, and chargebacks via push notification from Adyen&trade; triggered by external sources (for example, store)
```
